### PR TITLE
Adds Projects

### PR DESCRIPTION
**Pull request by toddsundsted**
I wasn't immediately sure if adding multiple projects was acceptable, but then I saw https://github.com/veelenga/awesome-crystal/pull/451 and thought, maybe so... I'm happy to break these up and/or leave anything out that's not of note or mature enough.

## Links
* https://github.com/toddsundsted/mxnet.cr
* https://github.com/toddsundsted/ishi
* https://github.com/toddsundsted/iterm2
* https://github.com/toddsundsted/public_suffix
* https://github.com/toddsundsted/host_meta
* https://github.com/toddsundsted/web_finger

## Checklist
* [-] - Shard is at least 30 days old*.
* [X] - Shard has CI implemented.
* [X] - Shard has daily/weekly periodic builds (ideally with Crystal latest and nightly).

All CI is on Travis (https://travis-ci.org/toddsundsted).

* Ishi and Iterm2 are currently only 18 days old, but both are an important part of the visualization work I do with MXNet, so they will both see their 30-day birthday. I'm happy to wait 12 more days and then merge this, if that's acceptable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -380,13 +384,15 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
...
  * [gopher.cr](https://github.com/anicholson/gopher.cr) - Gopher server toolkit
+ * [host_meta](https://github.com/toddsundsted/host_meta) - A Web Host Metadata (https://tools.ietf.org/html/rfc6415) client
  * [jwt](https://github.com/crystal-community/jwt) - Implementation of JWT (JSON Web Token)
  * [msgpack-crystal](https://github.com/crystal-community/msgpack-crystal) - MessagePack library
  * [OAuth](https://crystal-lang.org/api/OAuth.html) - OAuth consumer (Crystal stdlib)
  * [OAuth2](https://crystal-lang.org/api/OAuth2.html) - OAuth2 client (Crystal stdlib)
  * [OpenSSL](https://crystal-lang.org/api/OpenSSL.html) - bindings to libssl (Crystal stdlib)
  * [telnet.cr](https://github.com/spider-gazelle/telnet.cr) - Telnet protocol
  * [transfer_more](https://git.sceptique.eu/Sceptique/transfer_more) - Clone of transfer.sh to uploads files
+ * [web_finger](https://github.com/toddsundsted/web_finger) - A WebFinger (https://tools.ietf.org/html/rfc7033) client
```
Comment (veelenga) [inline README.md]: This can be moved to [Third-party APIs](https://github.com/veelenga/awesome-crystal#third-party-apis)

File: README.md [hunk 2]
```diff
@@ -380,13 +384,15 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [crystal-snmp](https://github.com/spider-gazelle/crystal-snmp) - An SNMP implementation with version 1, 2c and 3 support
  * [fast_irc.cr](https://github.com/RX14/fast_irc.cr) - Fast IRC parser/generator
  * [gopher.cr](https://github.com/anicholson/gopher.cr) - Gopher server toolkit
+ * [host_meta](https://github.com/toddsundsted/host_meta) - A Web Host Metadata (https://tools.ietf.org/html/rfc6415) client
```
Comment (veelenga) [inline README.md]: same

File: README.md [hunk 3]
```diff
@@ -358,13 +359,16 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
...
+ * [ishi](https://github.com/toddsundsted/ishi) - Graph plotting package with a small API powered by gnuplot
+ * [iterm2](https://github.com/toddsundsted/iterm2) - Display images within the terminal using the ITerm2 Inline Images Protocol
  * [lirith](https://github.com/lirith-engine/lirith) - Graphics engine
  * [m3u8](https://github.com/akiicat/m3u8) - Generate and parse m3u8 playlists for HTTP Live Streaming
  * [message_verifier.cr](https://github.com/danielwestendorf/message_verifier.cr) - Verify and generate Rails `ActiveSupport::MessageVerifier` signed tokens
  * [monads](https://github.com/alex-lairan/monads) - Monad implementation
  * [pangu.cr](https://github.com/isaced/pangu.cr) - Paranoid text spacing in Crystal
  * [pinger](https://github.com/spider-gazelle/pinger) - Ping IP addresses and DNS entries without requiring sudo
  * [port_midi](https://github.com/jimm/crystal_port_midi) - Crystal C bindings for the PortMIDI cross-platform MIDI I/O library
+ * [public_suffix](https://github.com/toddsundsted/public_suffix) - A small library designed to make the Public Suffix List (https://publicsuffix.org/) easier to use
```
Comment (veelenga) [inline README.md]: Maybe [Tools](https://github.com/veelenga/awesome-crystal#tools) ?

File: README.md [hunk 4]
```diff
@@ -358,13 +359,16 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [i18n.cr](https://github.com/vladfaust/i18n.cr) - Internationalization shard
  * [immutable](https://github.com/lucaong/immutable) - Implementation of thread-safe, persistent, immutable collections
  * [inflector.cr](https://github.com/phoffer/inflector.cr) - Singularize, pluralize, camelize, etc (port from ActiveSupport)
+ * [ishi](https://github.com/toddsundsted/ishi) - Graph plotting package with a small API powered by gnuplot
```
Comment (veelenga) [inline README.md]: What do you think about creating a new section for this entry? "Scientific", for instance?

File: README.md (hunk 1 above)
Comment (toddsundsted) [reply README.md]: will do

File: README.md (hunk 2 above)
Comment (toddsundsted) [reply README.md]: will do

File: README.md (hunk 3 above)
Comment (toddsundsted) [reply README.md]: okay will move

File: README.md (hunk 4 above)
Comment (toddsundsted) [reply README.md]: that sounds good to me!

Comment (toddsundsted) [reply README.md]: @veelenga how about moving it under "Science and Data analysis" since that already exists?

Comment (veelenga) [reply README.md]: Yeah, sounds good :+1:

File: README.md (hunk 1 above)
Comment (toddsundsted) [reply README.md]: fixed in b5b24f2

File: README.md (hunk 2 above)
Comment (toddsundsted) [reply README.md]: fixed in b5b24f2

File: README.md (hunk 3 above)
Comment (toddsundsted) [reply README.md]: fixed in b5b24f2

File: README.md (hunk 4 above)
Comment (toddsundsted) [reply README.md]: fixed in b5b24f2

**Pull request merged**